actors/market/tests/harness.rs

```diff
@@ -1150,6 +1150,9 @@ pub fn expect_query_network_info(rt: &MockRuntime) {
         quality_adj_power: power.clone(),
         pledge_collateral: TokenAmount::default(),
         quality_adj_power_smoothed: FilterEstimate::new(reward.atto().clone(), BigInt::zero()),
+        // TODO: flagging that this was set to zero, but this wouldn't be the actual return from the function
```

Comment:
Please remove this TODO. Zero is a reasonable value for networks other than mainnet.

Reply:
Done.